app/models/collection.rb

```diff
@@ -21,6 +21,13 @@ def accessioned?
     %w[first_draft depositing].exclude?(state)
   end
 
+  sig { returns(T.nilable(String)) }
+  def purl
+    return nil unless druid
+
+    File.join(Settings.purl_url, T.must(druid).delete_prefix('druid:'))
```

Comment:
Can you either add a test for this, or move this method to a module that Collection and Work share? I think the work already has a test for it.

Reply:
Done.